components/nexus-core/src/main/java/org/sonatype/nexus/internal/orient/DatabaseServerImpl.java

```diff
@@ -58,16 +65,24 @@
 {
   private final ApplicationDirectories applicationDirectories;
 
-  private final boolean listenerEnabled;
+  private final boolean binaryListenerEnabled;
+
+  private final boolean httpListenerEnabled;
+
+  private boolean dynamicPlugins;
 
   private OServer server;
 
   @Inject
   public DatabaseServerImpl(final ApplicationDirectories applicationDirectories,
-                            final @Named("${nexus.orient.listenerEnabled:-false}") boolean listenerEnabled)
+                            final @Named("${nexus.orient.binaryListenerEnabled:-false}") boolean binaryListenerEnabled,
+                            final @Named("${nexus.orient.httpListenerEnabled:-false}") boolean httpListenerEnabled,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)